api-participant/src/main/java/bio/terra/pearl/api/participant/controller/address/AddressValidationController.java

```diff
@@ -0,0 +1,32 @@
+package bio.terra.pearl.api.participant.controller.address;
+
+import bio.terra.pearl.api.participant.api.AddressValidationApi;
+import bio.terra.pearl.core.model.address.AddressValidationResultDto;
+import bio.terra.pearl.core.model.address.MailingAddress;
+import bio.terra.pearl.core.service.address.AddressValidationService;
+import bio.terra.pearl.core.service.address.AddressValidationServiceProvider;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+
+@Controller
+public class AddressValidationController implements AddressValidationApi {
+
+  ObjectMapper objectMapper;
+  AddressValidationService client;
+
+  AddressValidationController(
+      ObjectMapper objectMapper, AddressValidationServiceProvider clientProvider) {
+    this.client = clientProvider.get();
```

Comment:
Let's store the clientProvider, rather than the client.  This preserves the ability of the clientProvider to change the client even after startup
```suggestion
    this.clientProvider = clientProvider;
```